src/svgs/OpenEyeIcon.tsx

```diff
@@ -1,28 +1,44 @@
 import React from "react"
-import styled from "styled-components"
+
+interface OpenEyeIconProps {
+  fill?: string
+}
 
 /** Icon */
-export const OpenEyeIcon = props => {
+export const OpenEyeIcon: React.SFC<OpenEyeIconProps> = ({ fill = "#000" }) => {
   return (
-    <StyledOpenEye
-      width={props.width || 20}
-      height={props.height || 20}
+    <svg
+      width="18"
+      height="18"
+      viewBox="0 0 18 18"
       xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 12 12"
-      className={props.className}
     >
-      <path
-        fill="#c2c2c2"
-        d="M6,2A6.75,6.75,0,0,0,0,6a6.75,6.75,0,0,0,6,4,6.75,6.75,0,0,0,6-4A6.75,6.75,0,0,0,6,2ZM6,8.5A2.5,2.5,0,1,1,8.5,6,2.5,2.5,0,0,1,6,8.5Z"
-      />
-      <path
-        fill="#c2c2c2"
-        d="M6,4.5A1.5,1.5,0,1,0,7.5,6,1.5,1.5,0,0,0,6,4.5Z"
-      />
-    </StyledOpenEye>
+      <title>Action/View/Md</title>
+      <g
+        id="Action/View/Md"
+        stroke="none"
+        stroke-width="1"
+        fill="none"
+        fill-rule="evenodd"
+      >
+        <g
+          id="eye"
```

Comment:
Maybe remove these ids?

Reply:
They are coming from the original svgs and we are having those in all of our Icons here. I agree - we should maybe clean it up, but should probably do it in all the files here.

Reply:
Even if we don't clean everything up at once its best to think of this stuff as incremental, so removing now would require less effort in the future.